### ISSUE TITLE
INTULocationManager.sharedInstance().requestLocation never finishes

### ISSUE THREAD

**Issue by bogdan-razvan** [intuit/LocationManager#106]
Sometimes requestLocation never finishes, hanging in the air while not entering the completion block. Did anyone else stumble upon this problem?

**Comment by lwdupont** [comment]
What does "hanging in the air" mean?

When I was testing the latest version I released, I did see something like that happen, but it was when I was requesting a highly accurate location, but inside.. when I walked near a window, it almost immediately resolved. 

If you have a reproducible set of code, let me know?

**Comment by OliverPearmain-je** [comment]
I was having the same issue and discovered that with iOS 11 Apple have added the requirement of ANOTHER entry in your Info.plist `NSLocationAlwaysAndWhenInUseUsageDescription`. 

My issue was resolved when I added this key.  (Don't forget to update your InfoPlist.strings as well if you use one of those).

See here...
https://developer.apple.com/documentation/corelocation/choosing_the_authorization_level_for_location_services/requesting_always_authorization

and here...
https://stackoverflow.com/questions/44424092/location-services-not-working-in-ios-11
